AmericanMedicalAssociation/ama-style-guide-2
# [EWL-5062] SG2 | Correct Article Stub Metadata

## Ticket(s)

**Jira Ticket**
- [EWL-5062: SG2 | Correct Article Stub Metadata](https://issues.ama-assn.org/browse/EWL-5062)

## Description
Currently when adding json metadata information, but no paragraph to anything using the article stub molecule,  the metadata section does not appear. When data for this area is provided, the template should render it correctly.

This update refactors the article stub template removing overused macros and completely removes references and use of "title_only" which is a remnant of an old and abandoned requirement.

## To Test

- Pull `feature/EWL-5062-correct-article-stub-metadata` branch
- Run `gulp serve` in ama-style-guide-2/styleguide to ensure a local version of style guide is running.
- Navigate to [Pages > News](http://localhost:3000/?p=pages-news).
- Scroll to bottom of page and confirm 4-up article teaser has one article with metadata.
- Navigate to [Molecules > Article Stub](http://localhost:3000/?p=molecules-article-stub).
- Click on the gear icon in the top right corner of the screen and then click "show pattern info". There should be no reference to "title_only".

## Visual Regressions

When testing via backstop, multiple failures occur from some test cases containing the tools organism. This is due to text changes in the json. Bug ticket has been created, [EWL-5067 SG2 | Update Backstop references for Tools List](https://issues.ama-assn.org/browse/EWL-5067).

Reference images for the article stub backstop tests have been included in this PR.


## Relevant Screenshots/GIFs

### Four up teaser section of news page
![image](https://user-images.githubusercontent.com/4438120/40018077-d73a28cc-5780-11e8-886f-bdf393504420.png)


## Remaining Tasks
N/A


## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
